### PR TITLE
chore(docs): add mainnet

### PR DESCRIPTION
**Pull request by fabtreb**
Add mainnet

issue: none

### CODE REVIEW

File: docs/content/build/5-mainnet.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# Mainnet
```
Comment (ttarsi) [inline docs/content/build/5-mainnet.md:1]: This page is titled `5-mainnet`, but is 3rd in the sidebar (defined in screenshot below). Let's be consistent – the numbers in the title of the file are for convenience (documentation on how we use this [here](https://github.com/omni-network/omni/tree/main/docs/content), but feel free to update it).

Screenshot 2024-10-30 at 1 29 57 PM

File: docs/content/build/5-mainnet.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+| Ethereum | 1 | [See on ChainList](https://chainlist.org/chain/1) |
+| Arbitrum One | 42161 | [See on ChainList](https://chainlist.org/chain/42161) |
+| Optimism | 10 | [See on ChainList](https://chainlist.org/chain/10) |
+| Base | 8453 | [See on ChainList](https://chainlist.org/chain/8453) |
+
+### Resources
+
+| **Resource** | **Address** |
+| --- | --- |
+| Archive Node(s) RPC | Available upon request |
```
Comment (ttarsi) [inline docs/content/build/5-mainnet.md:32]: Let's just point them to quicknode

**Review by ttarsi — APPROVED** [review]
Approving with a couple comments

- Nit but I think now that we have `Mainnet`, it would look better to have `Testnet` and `Mainnet` for simplicity (rather than `Omega Testnet`)
Screenshot 2024-10-30 at 1 32 04 PM

File: docs/content/build/5-mainnet.md (hunk 1 above)
Comment (fabtreb) [reply docs/content/build/5-mainnet.md:1]: Bothered me too. I started updating the numbers but reverted to make the review easier, without deleted / new files. Suggest another PR to actually remove the numbering on the file names.

**Comment by fabtreb** [comment]
> Approving with a couple comments
> 
> * Nit but I think now that we have `Mainnet`, it would look better to have `Testnet` and `Mainnet` for simplicity (rather than `Omega Testnet`)

Agree, will update in separate PR

**Pull request merged**